playwright/src/main/java/com/microsoft/playwright/impl/Serialization.java

```diff
@@ -144,6 +148,14 @@ private SerializedValue serializeValue(Object value) {
         result.n = (Integer) value;
       } else if (value instanceof String) {
         result.s = (String) value;
+      } else if (value instanceof Date) {
+        result.d = ((Date)value).toInstant().toString();
+      } else if (value instanceof URL) {
+        result.s = ((URL)value).toString();
```

Comment:
this should be `result.u = ((URL)value).toString();` ? Let's add a test for this.